hera_pspec/tests/test_uvpspec.py

```diff
@@ -304,6 +304,43 @@ def test_clear(self):
         nt.assert_false(hasattr(uvp, 'Ntimes'))
         nt.assert_false(hasattr(uvp, 'data_array'))
 
+    def test_get_r_params(self):
+        uvp = copy.deepcopy(self.uvp)
+        baselines = [(24,25), (37,38), (38,39)]
+
+        rp = {'filter_centers':[0.],
+              'filter_widths':[250e-9],
+              'filter_factors':[1e-9]}
+
+        r_params = {}
+
+        for bl in baselines:
+            key1 = (0,) + bl + ('xx',)
+            key2 = (1,) + bl + ('xx',)
+            r_params[key1] = rp
+            r_params[key2] = rp
+
+        r_params_unique = {}
+        r_params_unique_bls = {}
+        r_params_index = -1
+        #build a compressed string to store r_params
```

Comment:
It would be more robust to create a new `UVPSpec` object from data that uses a set of `r_params`, and then check that those params have been stored successfully in the file. This tests seems to just add them manually to the `UVPSpec` object, which isn't the usual workflow.

Reply:
I have adjusted the testing methods that create `UVpspec` objects from data to include a realistic set of `r_params`.